Automattic/wp-calypso
# Add launch flow domain upsell

#### Changes proposed in this Pull Request

This change adds a new step to the launch-site flow with a 50/50 split test for English locales.

#### Testing instructions

0. Make sure you are using an English locale
1. Sign up for a free user and site
2. Launch the site from the /home menu
3. Select a free domain and plan
4. For users in the test group, the new step should be visible
5. Select a domain. The active button should launch the site and direct you to checkout
6. There's no need to create a new site to test the skip button but it might be a good idea anyway. Remove the cart items, go to http://calypso.localhost:3000/start/launch-site/domains-launch?siteSlug=YOURSITE.wordpress.com and on step 4 - skip
7. Assign yourself to the control group and repeat. No additional steps should appear

Here is how it looks right now:

<img width="715" alt="Screenshot 2020-12-03 at 16 36 10" src="https://user-images.githubusercontent.com/82778/101042371-ccedef80-3585-11eb-8a45-ff1826942693.png">

Note: it's currently being discussed whether this particular experiment could be run with ExPlat. There is currently no easy way to do it so it's likely going to use the old system but ExPlat folks have been pinged.
